index.html

```diff
@@ -12,6 +12,66 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+    <!--Navbar-->
+    <div class="container">
+        <div class="logo">
+          <img src="/img/karma-logo.svg" alt="logo">
+          </div>
+        <nav>
+          <ul>
+          <li><b>Meet Karma</b></li>
+          <li>How it Works</li>
+          <li>Store</li>
+          <li>Blog</li>
+          <li>Help</li>
+          <li>Login</li>
+          </ul>
+        </nav>
+    </div>
+
+        <div class="bckimg">
+            <br>
+            <br>
+            <br>
+            <br>
+            <p>Introducing Karma</p>
+            <p>Bring WiFi with you,everywhere you go.</p>
+            <button class="btn">Learn More</button>
+        </div>
+        <br>
+        <br>
+        <!--Services-->
+        <div class="services">
+            <p>Everyone needs a little Karma.</p>
```

Comment:
Is this body content or a title? If it's a title, it should use one of the header tags `h1`, `h2`, `h3` etc.